src/Language/PureScript/Scheme/CodeGen/Printer.hs

```diff
@@ -0,0 +1,27 @@
+module Language.PureScript.Scheme.CodeGen.Printer where
+
+import Data.Text                              (Text, pack, intercalate)
+import Language.PureScript.Scheme.CodeGen.AST (AST(..))
+
+showT :: Show a => a -> Text
+showT x = pack $ show x
```

Comment:
This function is usually called `tshow`, as the `T` postfix is usually used for monad transformers

```suggestion
tshow :: Show a => a -> Text
tshow = pack . show
```